app/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.9
+WORKDIR /usr/src/app
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+COPY . .
+EXPOSE 3000
+CMD ["python", "main.py"]
```

Comment:
sorry, this is the correct command, can you adjust 
'uvicorn app.main:app --port  8080 --env-file .env'